nms/app/packages/magmalte/app/components/feg/FEGGatewayDialog.js

```diff
@@ -204,6 +204,7 @@ export default function FEGGatewayDialog(props: Props) {
     served_network_ids: [],
     swx: {...getDiameterConfigs(swx)},
     csfb: {...getSCTPConfigs(csfb)},
+    s8: {},
```

Comment:
I think oriol fixed this issue. We shouldn't require passing in s8. Can you clean this up.